genesis/engine/entities/rigid_entity/rigid_entity.py

```diff
@@ -3017,6 +3020,92 @@ def is_local_collision_mask(self):
         """Whether the contype and conaffinity bitmasks of this entity only applies to self-collision."""
         return self._is_local_collision_mask
 
+    def sample(self, tet_cfg=None):
```

Comment:
can we put this function in IPC-related or standalone file? i.e. a function that takes a rigid entity and samples a tet mesh.